### PR TITLE
fedekunze/ Fixed stake tx history components

### PR DESCRIPTION
**Pull request by fedekunze**
`Redelegations` and `Unbondings` now show tokens instead of shares. 

You can pass the shares via the `sharesToTokens` prop or it calculates them for you based on the `calculateTokens` function from voyager if the amount is not given

screen shot 2018-10-01 at 1 07 44 pm

### CODE REVIEW

File: src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue [hunk 1]
```diff
@@ -106,9 +108,40 @@ export default {
...
+      let validator = this.validators.find(c => c.owner === validatorAddr)
+      return (validator && validator.description.moniker) || validatorAddr
+    },
+    prettyInt(amount) {
+      return numeral(amount).format(`0,0`)
+    },
+    pretty(amount) {
+      return numeral(amount).format(`0,0.00`)
+    },
+    calculatePrettyfiedTokens(validatorAddr, shares) {
```
[the remark this answers is not in the document]
Comment (fedekunze) [reply src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue]: addressed

File: src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue [hunk 2]
```diff
@@ -3,20 +3,17 @@ tm-li-transaction(:color="color" :time="transaction.time" :block="transaction.he
   template(v-if="delegation")
     div(slot="caption")
       | Delegated&nbsp;
-      b {{tx.delegation.amount}}
-      span &nbsp;{{tx.delegation.denom.toUpperCase()}}S
+      b {{ prettify(tx.delegation.amount) }}
+      span &nbsp;{{ bondingDenom }}s
```
Comment (jbibla) [inline src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue]: there should be an apostrophe if there is an added `s` 

...

`'s`

Comment (fedekunze) [reply src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue]: yes, the bonding denom is in singular. Although we should check the case when you're delegating only `1` and remove the `s`

Comment (fedekunze) [reply src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue]: edited

**Pull request merged**